app/actions/CustomActionBuilders.scala

```diff
@@ -31,30 +31,36 @@ class CustomActionBuilders(
 
   import CustomActionBuilders._
 
+  val membersIdentityClientId = "clientId" -> "members"
+
+  val recurringIdentityClientId = "clientId" -> "recurringContributions"
+
   // Tells identity to send users back to the checkout immediately after sign-up.
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
   // Prevents the identity validation email sending users back to our checkout.
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
-  private val idMember = "clientId" -> "members"
-
-  private def idWebAppRegisterUrl(path: String): String =
-    idWebAppUrl / "register" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & idMember
+  private def idWebAppRegisterUrl(path: String, clientId: (String, String)): String =
```

Comment:
Could `clientId` be a single String, since the key will presumably always be "clientId"?